src/pages/homePage/home.jsx

```diff
@@ -160,12 +161,11 @@ export const Home = () => {
                             </div>
                             <p className="product--downpayment">Zero downpayment</p>
                             <div className="product-cta">
-                                <a href="/cart_page/cart_page.html"> <button className="product__btn">Buy Now</button> </a>
                                 <img src={wishlist} alt="" className="product-wishlist" />
                             </div>
                         </div>
                     </div>
-                    <a href="/product_listing/product_listing.html"> <button className="btn-view-all">View all</button> </a>
+                    <NavLink to='/productListing'><button className="btn-view-all">View all</button> </NavLink>
```

Comment:
Use Link for simple link, if it is navigation link then NAVLink

Reply:
thank you, actually that line is deleted